plugins/src/ios/OptimoveSDKPlugin.swift

```diff
@@ -20,7 +20,11 @@ import NotificationCenter
             return nil
         }
         
-        let config = OptimoveConfigBuilder(optimoveCredentials: configValues[optimoveCredentialsKey], optimobileCredentials: configValues[optimoveMobileCredentialsKey])
+        let optimoveCredentials = configValues[optimoveCredentialsKey]?.isEmpty == true ? nil : configValues[optimoveCredentialsKey]
+        
+        let optimobileCredentials = configValues[optimoveMobileCredentialsKey]?.isEmpty == true ? nil : configValues[optimoveMobileCredentialsKey]
```

Comment:
is this some bug fix for Swift or why in android DDL pr?

Reply:
not sure if Barak worked on my branch or not, maybe it was one of the merges from dev, I will check that

Reply:
reverted, this is prob added in some other pr, mb error handling